code/obj/item/storage/secure_safe.dm

```diff
@@ -62,8 +62,8 @@ ABSTRACT_TYPE(/obj/item/storage/secure)
 		boutput(user, "You repair the lock on [src].")
 	return TRUE
 
-/obj/item/storage/secure/attackby(obj/item/W, mob/user, obj/item/storage/T)
-	if ((W.w_class > W_CLASS_NORMAL || istype(W, /obj/item/storage/secure)))
+/obj/item/storage/secure/attackby(obj/item/W, mob/user)
+	if ((!src.storage.check_can_hold(W) == STORAGE_CAN_HOLD || istype(W, /obj/item/storage/secure)))
```

Comment:
```suggestion
	if (!(src.storage.check_can_hold(W) == STORAGE_CAN_HOLD) || istype(W, /obj/item/storage/secure))
```
Needs parens so we're not just negating the result of the `check_can_hold` call

Reply:
https://github.com/goonstation/goonstation/pull/13023/commits/b6bf7e1e2bd3cbfc7c01ef7ab7164d2e952bfbb6